src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -388,6 +434,71 @@ unittest
     Test.test!("==")(interval.begin, 1554125192);
     /// 2019-04-01 15:26:31
     Test.test!("==")(interval.end, 1554132391);
+
+    /// Using time intervals with rounded hours
+    args = new Arguments;
+
+    setupTimeIntervalArgs(args, false);
+    args.parse("--time-interval 1h/h 1h");
+    interval = processTimeIntervalArgs(args);
+
+    timeToUnixTime("2019-04-01 14:00:00", dummy_time, dummy_conv);
+    Test.test!("==")(interval.begin, dummy_time);
+
+    timeToUnixTime("2019-04-01 14:59:59", dummy_time, dummy_conv);
+    Test.test!("==")(interval.end, dummy_time);
+
+    /// Using divided time intervals that start `now`
+    args = new Arguments;
+
+    setupTimeIntervalArgs(args, false);
+    args.parse("--time-interval now 1h/h");
+    interval = processTimeIntervalArgs(args);
+
+    timeToUnixTime("2019-04-01 15:26:32", dummy_time, dummy_conv);
+    Test.test!("==")(interval.begin, dummy_time);
+
+    timeToUnixTime("2019-04-01 15:59:59", dummy_time, dummy_conv);
+    Test.test!("==")(interval.end, dummy_time);
+
+    /// Using divided time intervals that ends `now`
+    args = new Arguments;
+
+    setupTimeIntervalArgs(args, false);
+    args.parse("--time-interval 1h/h now");
+    interval = processTimeIntervalArgs(args);
+
+    timeToUnixTime("2019-04-01 14:00:00", dummy_time, dummy_conv);
+    Test.test!("==")(interval.begin, dummy_time);
+
+    timeToUnixTime("2019-04-01 15:26:31", dummy_time, dummy_conv);
+    Test.test!("==")(interval.end, dummy_time);
+
+    /// Using divided `now`
+    args = new Arguments;
+
+    setupTimeIntervalArgs(args, false);
+    args.parse("--time-interval now/h now/m");
+    interval = processTimeIntervalArgs(args);
+
+    timeToUnixTime("2019-04-01 15:00:00", dummy_time, dummy_conv);
+    Test.test!("==")(interval.begin, dummy_time);
+
+    timeToUnixTime("2019-04-01 15:25:59", dummy_time, dummy_conv);
+    Test.test!("==")(interval.end, dummy_time);
+
+    /// Using one divided `now`
+    args = new Arguments;
+
+    setupTimeIntervalArgs(args, false);
+    args.parse("--time-interval now/h");
+    interval = processTimeIntervalArgs(args);
+
+    timeToUnixTime("2019-04-01 15:00:00", dummy_time, dummy_conv);
+    Test.test!("==")(interval.begin, dummy_time);
+
+    timeToUnixTime("2019-04-01 15:26:31", dummy_time, dummy_conv);
+    Test.test!("==")(interval.end, dummy_time);
```

Comment:
you have both tests for hour and minutes, one test with a `/d` (day) would also be good

Reply:
done